README.md

```diff
@@ -230,8 +232,11 @@ The following exceptions are thrown:
 
 * `RegistryAuthenticationException`: username and password are invalid
 * `RegistryAuthorizationException`: registry does not support pushing the layers or uploading the manifest using the given credentials, probably because the repository is private and the credentials provided do not have access
+* `RegisteryVersionException`: You're not running a registry that supports the v2 API.
```

Comment:
Please change this to "registry does not support the v2 API". The person running the lib here might not be running the registry; could be against a corporate or cloud registry.